src/server/controllers/VerificationController.ts

```diff
@@ -619,16 +619,48 @@ export default class VerificationController
       );
 
     this.router.route(["/verify/etherscan"]).post(
-      // TODO: add validation
+      body("address")
+        .exists()
+        .bail()
+        .custom(
+          (address, { req }) => (req.addresses = validateAddresses(address))
+        ),
+      body("chainId")
+        .optional()
+        .custom(
+          (chainId, { req }) =>
+            // Support both `body.chain` and `body.chainId`
+            (req.body.chain = chainId)
```

Comment:
why here why are not calling `checkChainId`?

Reply:
It's effectively checked at the next check `req.body.chain`. If double checked, the error sends double errors